src/transports/either.rs

```diff
@@ -20,10 +24,10 @@ impl<A, B, AOut, BOut> Transport for Either<A, B>
 where
     A: Transport<Out = AOut>,
     B: Transport<Out = BOut>,
-    AOut: futures::Future<Output = error::Result<rpc::Value>> + Unpin + 'static,
-    BOut: futures::Future<Output = error::Result<rpc::Value>> + Unpin + 'static,
+    AOut: futures::Future<Output = error::Result<rpc::Value>> + 'static + Send,
```

Comment:
I *think* previously EitherTransport did not require Send but now it does because of BoxFuture. I wasn't able to get the types to work out otherwise. Technically one could duplicate this code and use LocalBoxFuture to get rid of the Send requirement.

Reply:
IMHO acceptable trade-off for async code for now.